pyproject.toml

```diff
@@ -22,7 +22,7 @@ exclude_lines = [
 
 [tool.poetry]
 name = "cryptojwt"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
This seems like a minor version bump, 1.5.0 perhaps?

Reply:
OK